src/docs/ad/ad-introduction.rst

```diff
@@ -1,2 +1,40 @@
 Introduction to Active Directory Integration
-############################################
\ No newline at end of file
+############################################
+
+Direct Active Directory integration
+-----------------------------------
+
+Heterogeneous IT environments often contain various domains and operating systems that need to be able to seamlessly communicate. SSSD offers integration with Active Directory on Linux/Unix clients by taking advantage of the SSSD AD provider. In addition to retrieving and caching user and group information from Active Directory, SSSD can:
```

Comment:
In general, SSSD runs on Unix systems to some level but we, developers, have nothing to do with Unix. So perhaps it is better to omit it so we don't advertise a false assumption that we run tests against Unix systems?